### DISCUSSION TITLE
Test For Mac (pro m2)

### DISCUSSION THREAD

**Discussion by GondorGitaros** [lllyasviel/Fooocus#1182] [General]
HI! If you want some time test for mac i can make some. LMK if it's needed or not

**Comment by ShreyamMaity** [comment]
It would be great if you could help me out. I've been running some tests, but I can't seem to get below 114s per iteration.

**Comment by GondorGitaros** [comment]
Okey I will try and see if I can. I'm new to this GitHub thing is there something i should attach?

**Reply by ShreyamMaity** [reply]
Just share all the settings you explored and the sec per iteration score.

**Reply by GondorGitaros** [reply]
### **Settings**
extreme speed, fooocus photograph && default settings

### **(chatGPT made) prompt:**
Create a captivating portrait of a futuristic explorer standing at the edge of an otherworldly landscape, wearing an advanced space suit with intricate details. The explorer's expression should convey a sense of curiosity and wonder as they gaze into the unknown. Surround them with floating holographic displays showcasing alien flora and fauna. Illuminate the scene with soft, ethereal light, casting dynamic shadows on the explorer's face. Incorporate subtle futuristic elements to enhance the overall sci-fi atmosphere. The color palette should blend cosmic blues, purples, and greens to evoke a sense of mystery and adventure. 
	
### **Data**
moving model to gpu: 155mp
best sec per iteration: 109 sec

**Sorry for the late reply**

**Comment by jotalevi** [comment]
So, I just got a really good Sec per iteration, bout 19 on a base m2 Air using the same prompt as @GondorGitaros.
I changed some stuff in the code regarding mps / cpu usage and got it to work in bout a night. then I fucked everything up while trying to commit and lost some files. I'll try to PR a working commit tomorrow. I'm in LAS, might have a bit of a time ofset here
